assets/js/pages/Users/UsersPage.test.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+
+import MockAdapter from 'axios-mock-adapter';
+
+import { networkClient } from '@lib/network';
+import { screen, act, waitFor } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { adminUser, userFactory } from '@lib/test-utils/factories/users';
+import { renderWithRouter } from '@lib/test-utils';
+
+import UsersPage from './UsersPage';
+
+const axiosMock = new MockAdapter(networkClient);
+
+describe('UsersPage', () => {
+  afterEach(() => {
+    axiosMock.reset();
+  });
+
+  it('should render users page with a table without data.', async () => {
+    axiosMock.onGet('/api/v1/users').reply(200, []);
+    await act(async () => {
```

Comment:
We do you need this `act`?
This test doesn't really look an `async` test, bu well, maybe I'm wrong :sweat_smile:

Reply:
I thought we should use act when the state changes, played around it fails without xD

Reply:
I still think we don't need it, and there are ways to do it without (like useing `findByText` which has a default timeout), but well, it is alright